src/cli/utils.ts

```diff
@@ -91,3 +91,34 @@ export function envPaths (name: string) {
         temp: path.join(tmpdir, username, name),
     };
 }
+
+export function invalidSubdomain(subDomain: string) {
```

Comment:
Is the playpass CLI the right place for this validation? Shouldn't this be server side?